README.md

```diff
@@ -92,13 +137,13 @@ to ensure that CI workloads run in staging cannot interfere with production reso
 ### I have an existing ML project. Can I productionize it using MLOps Stacks?
 Yes. Currently, you can instantiate a new project and copy relevant components
 into your existing project to productionize it. MLOps Stacks is modularized, so
-you can e.g. copy just the GitHub Actions workflows under `.github` or ML resource configs
+you can e.g. copy just the GitHub Actions workflows under `.github` or ML asset configs
  under ``{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/resources`` 
 and ``{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/databricks.yml`` into your existing project.
 
 ### Can I adopt individual components of MLOps Stacks?
 For this use case, we recommend instantiating via [Databricks asset bundle templates](https://docs.databricks.com/en/dev-tools/bundles/templates.html) 
-and copying the relevant subdirectories. For example, all ML resource configs
+and copying the relevant subdirectories. For example, all ML asset configs
```

Comment:
Hmm should we rename the associated folder then to `assets`? I think we chose resources because of Terraform & DAB resources, so it might be better to use that for consistency over assets

Reply:
Synced offline, good to go on `assets` rename